main/src/main/scala/sbt/internal/Act.scala

```diff
@@ -242,26 +239,15 @@ object Act {
   def toAxis[T](opt: Option[T], ifNone: ScopeAxis[T]): ScopeAxis[T] =
     opt match { case Some(t) => Select(t); case None => ifNone }
 
-  def config(confs: Set[String]): Parser[ParsedAxis[String]] = {
-    val sep = ':' !!! "Expected ':' (if selecting a configuration)"
-    token(
-      (ZeroString ^^^ ParsedZero | value(examples(ID, confs, "configuration"))) <~ sep
-    ) ?? Omitted
-  }
-
   // New configuration parser that's able to parse configuration ident trailed by slash.
   private[sbt] def configIdent(
```

Comment:
Nit: Since `config` is now gone, it's a good opportunity to name this back to `config` instead of using what seems like a differentiated method name.

Reply:
`config` has traditionally meant the lower case configuration name like `compile`, whereas `configIdent` was created to mean the Scala identity name, the upper case `Compile`. See also `zeroIdent`, `thisBuildIdent`, etc. It's a few more characters, but I feel more comfortable keeping this around since a lot of these are just passing string around and easy to get mixed up.